faucet.go

```diff
@@ -725,11 +727,60 @@ func (l *lightningFaucet) invoicePage(w http.ResponseWriter, r *http.Request) {
 	http.Error(w, "", http.StatusMethodNotAllowed)
 }
 
+func (l *lightningFaucet) createUpdatesChans(id string) (
+	chan lnrpc.Invoice_InvoiceState, chan string, error) {
+	ctx := context.Background()
```

Comment:
You need to pass in `ctx` in the arguments of `createUpdatesChan`:

```go
func (l *lightningFaucet) createUpdatesChans(ctx context.Context, id string) ( ...
```

(see next inline comment)